src/current/cockroachcloud/plan-your-cluster-serverless.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
because most dedicated docs were created first, most of our serverless vs dedicated page naming is of the structure `page-name` for dedicated and `page-name-serverless` or `serverless-page-name` for serverless. i agree this should be improved but i'd rather open a new pr than change more page names here. this has been on the back of my mind for a while, though, so thank you for bringing it up!